starter-code/server.js

```diff
@@ -36,13 +40,17 @@ app.use(express.static('./public'));
 app.get('/new', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
   // Put your response here...
+  // From the prespective of the server the follow line only corresponds to #5 because that line only send the file thus responding. This doesn't really correspond to any code in article.js because this query serves up the new.html page rather than updating the actual articles within that page. This doesn't use any of the CRUD functions that are used in the actual database as it is only serving up the page.
+  //
+  //
   response.sendFile('new.html', {root: './public'});
 });
 
 // REVIEW: Routes for making API calls to use CRUD Operations on our database
 app.get('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
```

Comment:
.5 - (server.js) ln. 44 “GET: Articles routes for getting ALL articles from the database, and returning to client” - This does have a place in CRUD